callbacks/control_bar.py

```diff
@@ -39,8 +39,7 @@
 
 # Create an empty file if it doesn't exist
 if not os.path.exists(EXPORT_FILE_PATH):
-    with open(EXPORT_FILE_PATH, "w") as f:
-        pass
+    open(EXPORT_FILE_PATH, "w")
```

Comment:
Change this to `open(EXPORT_FILE_PATH, "w").close()` so that the open connection doesn't leak memory